ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -568,6 +566,45 @@ def test_paddle_tensor_add_(
     )
 
 
+# addmm
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="addmm",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"), num_arrays=3, shared_dtype=True
+    ),
+    alpha=helpers.floats(min_value=0.01, max_value=1),
+    beta=helpers.floats(min_value=0.01, max_value=1),
```

Comment:
@illia-bab I've added functions to generate values for testing alpha and beta, I'm also generating the tensor values with `dtype_and_values` instead

Reply:
@asuzukosi great changes! Make sure to let me know once you test the implementation and make sure it works as expected! Thanks!